CodeEdit/Features/AppPreferences/Sections/GeneralPreferences/GeneralPreferencesView.swift

```diff
@@ -162,6 +165,24 @@ private extension GeneralPreferencesView {
         }
     }
 
+    var reopenAfterWindowCloseBehaviourSection: some View {
+        PreferencesSection("After last window closed") {
+            Picker(
+                "After last window closed:",
+                selection: $prefs.preferences.general.reopenWindowAfterClose
+            ) {
+                Text("Do nothing")
+                    .tag(AppPreferences.ReopenWindowBehavior.doNothing)
+                Divider()
```

Comment:
Is a divider useful here with only three items? It may be, just wanted feedback on this.

Reply:
if you want i can get it removed but personally i think it looks better

Reply:
I actually like the idea to separate the default option like this. Gives a hint what the default is if you come back later